examples/example-gateway/gateway.json

```diff
@@ -7,5 +7,6 @@
 
 	"clientConfig": "./clients",
 	"endpointConfig": "./endpoints",
-	"middlewareConfig": "./middlewares/middleware-config.json"
+	"middlewareConfig": "./middlewares/middleware-config.json",
+	"copyrightHeader": "./copyright_header.txt"
```

Comment:
Putting this in gateway.json seems to "leak" zanzibar responsibility to user.

We can just have one copyright that we default to.